Core/clim-core/panes/stream-panes.lisp

```diff
@@ -368,61 +370,88 @@ current background message was set."))
-  (apply #'make-clim-stream-pane :type 'application-pane options))
-
-(defun make-clim-pointer-documentation-pane (&rest options)
-  (apply #'make-clim-stream-pane :type 'pointer-documentation-pane options))
-
-(defun make-clim-command-menu-pane (&rest options)
-  (apply #'make-clim-stream-pane :type 'command-menu-pane options))
+  (when display-after-commands-p
+    (check-type display-after-commands (member nil t :no-clear))
+    (when (member :display-time options)
+      (error "MAKE-CLIM-STREAM-PANE can not be called with both
+    :DISPLAY-AFTER-COMMANDS and :DISPLAY-TIME keywords")))
+  (let* ((other-options (alexandria:remove-from-plist options :type :scroll-bar
+                                                              :scroll-bars :borders
+                                                              :label :label-alignment
+                                                              :display-after-commands))
+         ;; See Spec 29.4.3 for :display-after-commands keyword.
+         (final-options (if display-after-commands-p
+                            (list* :display-time
+                                   (if (eq display-after-commands t)
+                                       :command-loop
+                                       display-after-commands)
+                                   other-options)
+                            other-options))
+         (pane (apply #'make-pane type final-options))
+         (stream pane))
+    (when scroll-bars
+      (setq pane (apply #'make-pane 'scroller-pane
+                        :contents (list (make-pane 'viewport-pane
+                                                   :contents (list pane)))
+                        ;; From the Franz manual if :scroll-bars is a
+                        ;; cons the car is treated as the non-cons
+                        ;; argument and the cdr is a list of keyword
+                        ;; argument pairs to be used as options of
+                        ;; the scroller-pane
+                        (if (consp scroll-bars)
+                            `(:scroll-bar ,@scroll-bars)
+                            `(:scroll-bar ,scroll-bars)))))
+    (when label
+      (setq pane (apply #'make-pane 'label-pane
+                        :label label
+                        :contents (list pane)
+                        (when label-alignment-p
+                          (list :label-alignment label-alignment)))))
+    (when borders
+      (setq pane (apply #'make-pane 'outlined-pane
+                        :thickness (if (not (numberp borders))
+                                       1
+                                       borders)
+                        :contents (list pane)
+                        nil)))
+    (values pane stream)))
+
+(defun make-clim-interactor-pane (&rest options &key (scroll-bar t scroll-bar-p) (scroll-bars scroll-bar scroll-bars-p) &allow-other-keys)
+  (declare (ignorable scroll-bars))
```

Comment:
Please use `ignore` instead of `ignorable`. The latter is for macros which include unknown code in the expansion that might or might not use a given variable.